config/config.go

```diff
@@ -59,6 +62,7 @@ func DefaultConfig() Config {
 		P2P:        p2pConfig.DefaultConfig(),
 		API:        apiConfig.DefaultConfig(),
 		CONSENSUS:  consensusConfig.DefaultConfig(),
+		TimeSync:   timeSyncConfig.DefaultConfig(),
```

Comment:
It'll be better if we follow the Caps convention and rename to something like `TIME`. you're using the right case, but let's wait with it until we refactor everything to be the same.

Reply:
Changed to TIME as requested